chapter07/sxg/ex_7_5.pl

```diff
@@ -0,0 +1,20 @@
+#!perl
+use warnings;
+use strict;
+
+my $find=0;
+my $line;
+
+print "I found:\n";
+while (<>) {
+	$line = $_;
+
+  	if (/[A-Z][a-z]+/) {

```

Comment:
Does this + have an effect as there is no requirement for what is either side of the expression?